### ISSUE TITLE
Tailwind integration into Phoenix Framework - watch not rebuilding changes to content

### ISSUE THREAD

**Issue by cvkmohan** [parcel-bundler/parcel#7634]
I was trying to integrate Parecel2 along with ParcelCSS as the asset management solution for Phoenix framework.
The integration went smooth and the performance looks very nice as well. However, I ran into an issue with watch not rebuilding the css on changes to content.
In fact, my same issue has been documented very clearly here https://github.com/tailwindlabs/tailwindcss/issues/7136 - it has been closed as well. The solution seems to be ```postcss-cli``` upgrade to include ```dir-dependency```
Now, I am facing the same issue. However, parcel-css says we can avoid postcss. So, how to fix this problem?

**Comment by mischnic** [comment]
Please provide a reproduction

**Comment by cvkmohan** [comment]
@mischnic, thanks for the response. I did not get the notification - hence the delay in response.
I could resolve the issue after moving parcel config into the project root folder. As you might be aware, in Phoenix asset management configuration resides in a folder called assets. I could not make parcel work by installing in that folder. However, when I moved the node_modules folder and package.json file into the project root folder, parcel watch task started working correctly.
I am closing this issue now. However, I would be curious to know, how to make parcel watch set of files/folders explicitly.
